pages/en/blog/release/v21.1.0.md

```diff
@@ -1,9 +1,9 @@
 ---
-date: 2023-10-24T13:54:53.218Z
+date: '2023-10-24T13:54:53.218Z'
 category: release
 title: Node v21.1.0 (Current)
 layout: blog-post.hbs
-author: Michaël Zasso
+author: 'Michaël Zasso'
```

Comment:
It's not clear if my name will be quoted in future release posts.

Reply:
It shouldn't; that's a good catch. I forgot to add special alphabetic characters to the regex. Ahem.

Reply:
Fixed.